src/Tizen.NUI/src/internal/Interop/Interop.TextEditor.cs

```diff
@@ -235,6 +235,9 @@ internal static partial class TextEditor
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_TextEditor_SelectWholeText")]
             public static extern void SelectWholeText(global::System.Runtime.InteropServices.HandleRef jarg1);
 
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_TextEditor_SelectText")]
+            public static extern void SelectText(global::System.Runtime.InteropServices.HandleRef jarg1, uint jarg2, uint jarg3);
+
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_TextEditor_Property_SELECTED_TEXT_get")]
```

Comment:
Like the comment at the csharp binder patch,
I think these jarg* name could be more understandable name.